builders/interoperability/xcm/xc-registration/.pages

```diff
@@ -4,3 +4,4 @@ nav:
   - 'Forum Templates': 'forum-templates.md'
   - 'XC Channel Registration': 'xc-integration.md'
   - 'XC Asset Registration': 'assets.md'
+  - 'Foreign Asset Registration': 'foreign-asset-registration.md'
```

Comment:
How does this guide relate to XC Asset Registration? The distinction might not be clear to users deciding which page to go to

Reply:
Good point! I clarified the titles and descriptions to differentiate them more.